src/main/java/tests/enterZipCodeTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import domain.pages.DirectoryPage;
+import domain.pages.HomePage;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class enterZipCodeTest {
+
+    @Test
+            public void isZipCodeCorrect()
```

Comment:
Please add a description of what the test does
update method name accordingly...